FusionIIIT/applications/Purchase_store/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import User
+from applications.globals.models import Staff
+from applications.filetracking.models import File
+
+class Indent_File(models.Model):
```

Comment:
Class Names should be in Pascal Case. Don't mix snake case with Pascal Case(follow PEP-8).
Please, add docstring to each model's class.